ta/pkcs11/src/object.c

```diff
@@ -1051,6 +1056,11 @@ enum pkcs11_rc entry_set_attribute_value(struct pkcs11_client *client,
 	if (rc)
 		goto out;
 
+	/* Set key check value attribute */
+	rc = set_check_value_attr(&head);
```

Comment:
@etienne-lms I have a concern here, as per your recommandation  `set_check_value_attr()` has been moved before  `check_attrs_against_modification()` to santize `CKA_CHECK_VALUE ` against modification, however, at this stage the object is object attributes are required to compute the KCV such as the class and key type. Here, the template and object are both needed to compute/test the KCV. My question, should this call moved elsewhere where all attributes are consolidated/merged together ?